packages/widget/src/components/common/buttons/connect-wallet.ts

```diff
@@ -62,49 +60,52 @@ export class ConnectWalletButton extends BaseComponent {
     return !!this.wallets.evmWallet || !!this.wallets.substrateWallet;
   }
 
-  render(): HTMLTemplateResult {
+  private renderConnectWalletButton(): HTMLTemplateResult | undefined {
+    if (this.wallets.substrateWallet) return;
```

Comment:
why do we need this line?
 If you remove it, it will render disconnect button that can be used to disconnect wallet (both evm and substrate)

Reply:
@mpetrunic I implemented it as we have it in a substrate design dropdown. IMO it looks nicer and as I already spent time implementing the feature, it makes sense to keep it. You should see the design in the last deployment, as I removed the filter for non evm

Reply:
Are you sure you've commited removal of filter?
![image](https://github.com/sygmaprotocol/sygma-widget/assets/8836210/5baf6f40-b6ee-4f5d-bb59-2512b699be05)


I was trying to say lets keep both inside dropdown and outside?

Reply:
Last deployment https://7e58957d.sygma-react-widget.pages.dev/


<img width="401" alt="Screenshot 2024-02-22 at 12 29 28" src="https://github.com/sygmaprotocol/sygma-widget/assets/8141917/f96e6149-a680-43cf-95b2-a9f49ecdf81b">

Reply:
Last deployment is https://aed97e49.sygma-widget.pages.dev/ 45 mins ago?!

Reply:
@mpetrunic as a substrate selector is bigger, it doesn't fit well without additional changes. I think we can better work on the evm selector soon to align the design